MANUAL.md

```diff
@@ -29,6 +29,36 @@ Cloud Foundry.
     assign a pushed application that did not specify its memory
     requirements, explicitly.  Defaults to `256`.
 
+  - `default_app_disk_in_mb` - How much disk (in megabytes) to
+    assign a pushed application that did not specify its memory
+    requirements, explicitly. Defaults to `1000`.
+
+  - `uaa_lockout_failure_count` - Amount of failed UAA login attempts
+    before lockout.
```

Comment:
"Number" works better here, since "amount" has connotation of being large (amount of money) or even uncountable (amount of good will).